third_party/txt/src/minikin/GraphemeBreak.h

```diff
@@ -17,6 +17,9 @@
 #ifndef MINIKIN_GRAPHEME_BREAK_H
 #define MINIKIN_GRAPHEME_BREAK_H
 
+#include <stddef.h>
+#include <unicode/utf16.h>
```

Comment:
Where is this include needed?

Reply:
Using size_t and uint16_t constructs. Not available without these imports. Internally we have checks that make sure each header can be parsed/compiled individually.

If this is still weird, let's chat.